index.html

```diff
@@ -0,0 +1,124 @@
+            <li><a class="nav-link" href="#something">Features</a></li>
+            <li><a class="nav-link" href="#somethingmore">How It Works</a></li>
+            <li><a class="nav-link" href="#evenmore">Prices</a></li>
+          </ul>
+        </nav>
+      </header>
+      <div>
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+          <input
+            name="email"
+            id="email"
+            type="email"
+            placeholder="Enter your email address"
+            required=""
+          />
+          <input name="submit" id="submit" type="submit" value="Submit" />
+        </form>
+      </div>
+
+      <section id="something">
+        <div>
+          <div class="icon">
+            <img
+              src="https://cdn-icons-png.flaticon.com/512/1350/1350727.png"
+              alt="Materials"
+              width="60"
+              height="70"
+            />
+          </div>
+          <div class="descrip">
+            <h2>Premium Materials</h2>
+            <p>
+              Our trumpets use the shiniest brass which is sourced locally. This
+              will increase the longevity of your purchase.
+            </p>
+          </div>
+        </div>
+        <div>
+          <div class="icon">
+            <img
+              src="https://cdn-icons-png.flaticon.com/512/5770/5770775.png"
+              alt="Materials"
+              width="70"
+              height="70"
+            />
+          </div>
+
+          <div class="descrip">
+            <h2>Fast Shipping</h2>
+            <p>
+              We make sure you recieve your trumpet as soon as we have finished
+              making it. We also provide free returns if you are not satisfied.
+            </p>
+          </div>
+        </div>
+        <div>
+          <div class="icon">
+            <img
+              src="https://cdn-icons-png.flaticon.com/512/3500/3500954.png"
+              alt="Materials"
+              width="60"
+              height="60"
+            />
+          </div>
+          <div class="descrip">
+            <h2>Quality Assurance</h2>
+            <p>
+              For every purchase you make, we will ensure there are no damages
+              or faults and we will check and test the pitch of your instrument.
+            </p>
+          </div>
+        </div>
+      </section>
+      <section id="somethingmore">
+        <div>
+          <iframe
+            width="420"
+            height="315"
+            src="https://www.youtube.com/embed/nODd-8rvJJU"
+          >
+          </iframe>
+        </div>
+      </section>
+      <section id="evenmore">
+        <div>Price1</div>
+        <div>Price2</div>
+        <div>Price3</div>
+      </section>
+      <footer>
+        <ul>
+          <li><a href="#">Privacy</a></li>
+          <li><a href="#">Terms</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+        <span>Copyright 2022, Original Trumpets</span>
+      </footer>
```

Comment:
[OPTIONAL] Please I suggest the `footer` element should be used to provide a logical structure to the page and include an `aria-label` attribute to indicate that it is the page `footer` for screen readers.